adot/python/src/otel/otel_sdk/adot-instrument

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
Do we need to change the name of this file?

Reply:
Rename might be an indication that user is using adot lambda layer but not upstream one.

Reply:
adot-instrument add xray propagator

Reply:
Is it helpful for the user? I guess if we use the same name, then some random document they find on Google can be applied regardless of if it's about upstream or downstream.